test/pkg/environment/expectations.go

```diff
@@ -125,6 +128,31 @@ func (env *Environment) eventuallyExpectScaleDown() {
 	}).Should(Succeed(), fmt.Sprintf("expected scale down to %d nodes, had %d", env.Monitor.NodeCountAtReset(), env.Monitor.NodeCount()))
 }
 
+func (env *Environment) GetCreatedNodes(before []v1.Node, after []v1.Node) []*v1.Node {
```

Comment:
Can we collapse this and ExpectCreatedNodeCount? The count should be the length of the list.